shared/templates/template_IGNITION_service_disabled

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+# reboot = true
```

Comment:
Is a reboot needed?

Reply:
Yes. It happens whenever you add/modify a MachineConfig. The machine-config-operator does that.